### PR TITLE
Add vertical scroll bar to ScrollViews

### PR DESCRIPTION
**Pull request by vitvly**
Small fix for some kind of scrollbar to appear.

### CODE REVIEW

**Review by maxhora — CHANGES_REQUESTED** [review]
I believe that scrollbars should be optional, probably, controlled with prop in JS.
@vkjr what do you think?

File: ReactQt/runtime/src/qml/ReactScrollListView.qml [hunk 1]
```diff
@@ -53,4 +53,8 @@ ListView{
             }
         }
     }
+
+    ScrollBar.vertical: ScrollBar {
+        width: 10
```
Comment (MaxRis) [inline ReactQt/runtime/src/qml/ReactScrollListView.qml]: It should also has great sense to make width customizable

**Comment by vkjr** [comment]
@maxris, @siphiuel, ScrollView has `showsVerticalScrollIndicator` `showsHorizontalScrollIndicator` properties that can be used.
By default these properties set to `true`, but scrollbars visible only for couple seconds while interaction in progress. When dragging stops, they disappear. I believe it would be nice to have same behavior. @siphiuel, does qt scrollbar behaves similarly? And I'm wondering if it has some default width value? I also would be glad not to specify a fixed width but use default instead.

**Comment by siphiuel** [comment]
@vkjr - yes, by default it's the same behaviour. There is a `policy` property (https://doc.qt.io/qt-5.11/qml-qtquick-controls2-scrollbar.html#policy-prop) that can be modified to change scroll bar behavior.

**Comment by siphiuel** [comment]
@vkjr - yes, there is a default width value, so it can work with just `ScrollBar.vertical: ScrollBar {}`.

**Comment by siphiuel** [comment]
Added support `showsHorizontalScrollIndicator` and `showsVerticalScrollIndicator` properties, similarly to stock ReactNative. Scrollbar `width` is not available for customization in iOS/Android versions, so maybe we can skip on implementing it.

**Review by vkjr — APPROVED** [review]

**Review by maxhora — APPROVED** [review]

**Comment by siphiuel** [comment]
@vkjr Fixed. Thank you for patience :)

**Pull request merged**